gen/dcos-config.yaml

```diff
@@ -379,6 +379,31 @@ package:
         ],
         "enabled": "{{ telemetry_enabled }}"
       }
+  - path: /etc/dcos-metrics-config.yaml
+    content: |
+      #######################################
+      # Configuration file for dcos-metrics #
+      #######################################
+
+      # Collector configuration
+      collector:
+        http_profiler: false
+
+        node:
+          poll_period: 15
+
+        # DC/OS (Mesos) agent configuration
+        mesos_agent:
```

Comment:
For this also: What would this all look like if I were running a master and agent on the same host? (Something which is planned for the future)